cli/cryptopia.go

```diff
@@ -20,8 +21,10 @@ func submitWithdrawCMD() cli.Command {
 			if c.NArg() < 3 || c.NArg() > 4 {
 				return errInvalidInput
 			}
-			var err error
-			amount := 0.0
+			var (
+				err    error
+				amount float64
+			)
```

Comment:
typically an `if foo, err := bar(); err != nil` pattern is used when using `:=`,  if the `foo` variable is needed, then the pattern would be
```go
foo, err := bar()
if err != nil {
    return err
}
...
```